velox/docs/functions/spark/math.rst

```diff
@@ -66,6 +66,26 @@ Mathematical Functions
     Returns ``x`` rounded up to the nearest integer.  
     Supported types are: BIGINT and DOUBLE.
 
+.. function:: check_add(x, y) -> [same as x]
+
+    Returns the result of adding x to y. The types of x and y must be the same and integral types.
+    Overflow results in an error.
```

Comment:
I notice the add function is also described as `overflow results in an error`. Shall we illustrate the difference between them?

> spark:function:: add(x, y) -> [same as x]
    Returns the result of adding x to y. The types of x and y must be the same.
    For integral types, overflow results in an error. Corresponds to sparks's operator ``+``.

Reply:
Good catch, I think this is not accurate, overflow would not result the error for `add` function.  Let me correct it also.
![image](https://github.com/facebookincubator/velox/assets/10524738/ccdeb4d9-1d7f-4f6b-a2ba-1b74736c5b21)

Reply:
Updated, thanks.